docs/xls-33d-multi-purpose-tokens/reference/mptoken.md

```diff
@@ -44,8 +44,8 @@ The `MPTokenID` is the result of SHA512-Half of the following values, concatenat
 | LedgerEntryType   | number    | UInt16        | The value 0x007F, mapped to the string `MPToken`, indicates that this object describes an individual account's holding of an MPT. |
 | `Account`           | string    | AccountID     | The owner of the MPT. |
 | `MPTokenIssuanceID` | string    | UInt256       | The `MPTokenIssuance` identifier. |
-| `MPTAmount`         | string    | UInt64        | This value specifies a positive amount of tokens currently held by the owner. Valid values for this field are between 0x0 and 0xFFFFFFFFFFFFFFFF. |
-| `LockedAmount`      | string    | UInt64        | (Default) Specifies a positive amount of tokens that are currently held in a token holder's account but that are unavailable to be used by the token holder. Locked tokens might, for example, represent value currently being held in escrow, or value that is otherwise inaccessible to the token holder. The initial value is 0 in order to save space on the ledger for an empty MPT holding. |
+| `MPTAmount`         | string    | Base10        | This value specifies a positive amount of tokens currently held by the owner. Valid values for this field are between 0 and 18,446,744,073,709,551,615. |
```

Comment:
```suggestion
| `MPTAmount`         | string    | Base10        | This value specifies a positive amount of tokens currently held by the owner. Valid values for this field are between 0 and 9,223,372,036,854,775,807. |
```

Should be the largest 63 bit integer, which is 9,223,372,036,854,775,807. Apologies this might due to a typo back then in the spec.